pb_nautobot_install.yml

```diff
@@ -79,8 +79,15 @@
         dest: "/root/.nautobot/nautobot_config.py"
         mode: 0644
 
-    - name: "RUN MAKEMIGRATIONS"
-      ansible.builtin.command: "/usr/local/bin/nautobot-server makemigrations"
+    # - name: "RUN MAKEMIGRATIONS"
+    #  ansible.builtin.command: "/usr/local/bin/nautobot-server makemigrations"
 
-    - name: "RUN MIGRATE"
-      ansible.builtin.command: "/usr/local/bin/nautobot-server migrate"
+    # - name: "RUN MIGRATE"
+    #  ansible.builtin.command: "/usr/local/bin/nautobot-server migrate"
+
+    - name: "LOAD NAUTOBOT DATA INTO DATABASE"
+      ansible.builtin.command: "/opt/nautobot/templates/loaddata.sh"
+      environment:
+        PGUSER: "{{ lookup('env', 'DB_USER') }}"
+        PGPASSWORD: "{{ lookup('env', 'DB_PASSWORD') }}"
+        PGDATABASE: "{{ lookup('env', 'DB_NAME') }}"
```

Comment:
Should probably add a `when: "{{ lookup('env', 'LOAD_DUMP_DATA') }}" to this, adding as a default yes, but a way to exclude it.

Reply:
I think the point of the lab is to have users execute `docker run...` and not have to build the container.  In which case I don't really see the point of having options via environment variables that only impact the container build.

Reply:
That's a good point. So the conditionals need to be moved to another place. Or are we looking at multiple container images?

- AIO empty
- AIO Preloaded
- AIO All the Plugins loaded
- AIO Onboarding only

??